src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -178,4 +180,20 @@ export default {
 	top: 12px;
     left: 8px;
 }
+
+.zoom-enter-active {
+  animation: zoom-in .1s;
+}
+.zoom-leave-active {
+  animation: zoom-in .1s reverse;
```

Comment:
You can also use `will-change` to optimize the browser rendering :)
`will-change: transform`

Reply:
https://medium.com/outsystems-experts/how-to-achieve-60-fps-animations-with-css3-db7b98610108

Reply:
Thanks for the tip :)